cms/djangoapps/cms_user_tasks/tasks.py

```diff
@@ -21,7 +21,7 @@
 
 @shared_task(bind=True)
 @set_code_owner_attribute
-def send_task_complete_email(self, task_name, task_state_text, dest_addr, detail_url):
+def send_task_complete_email(self, task_name, task_state_text, dest_addr, detail_url, olx_validation_text):
```

Comment:
we can maybe set default value of `olx_validation_text` to `None`. So we dont have to add this parameter everytime we call this method.